### ISSUE TITLE
Publish Azurite dotnet tool

### ISSUE THREAD

**Issue by psibernetic** [Azure/Azurite#109]
The dotnet CLI now has support for both local and global tools which should be considered the first-class tooling experience for C#. 

This would provide a superior experience for C# developers that do not otherwise use NPM and do not benefit from the NuGet package since init.ps1 is not supported in dotnetcore.

**Comment by XiaoningLiu** [comment]
@psibernetic  Good idea. Currently, we are lacking resources to handle plenty of Azurite issues and new architecture things. Will track the issue here for further investigation.
